test/adapter.spec.js

```diff
@@ -96,6 +105,8 @@ describe('adapter', () => {
     expect(adapter.getObserver(foo, 'bar', Object.getPropertyDescriptor(foo, 'bar'))).not.toBe(null);
     expect(adapter.getObserver(foo, 'baz', Object.getPropertyDescriptor(foo, 'baz'))).toBe(null);
     expect(adapter.getObserver(foo, 'xup', Object.getPropertyDescriptor(foo, 'xup'))).not.toBe(null);
+    expect(adapter.getObserver(foo, 'strict1', Object.getPropertyDescriptor(foo, 'strict1'))).not.toBe(null);
+    expect(adapter.getObserver(foo, 'strict2', Object.getPropertyDescriptor(foo, 'strict2'))).not.toBe(null);
```

Comment:
You're right, I should have thought of that. It's been some time since the PR; when I have the time again, I will see that I do the change, update my setup and run the tests again (even if it's a trivial change).